### PR TITLE
LÖVE 11.3 update

### PR DESCRIPTION
**Pull request by Veslo5**
Changed run loop by LÖVE latest 11.0 run callback.
https://love2d.org/wiki/love.run

### CODE REVIEW

File: main.lua [hunk 1]
```diff
@@ -8,60 +8,45 @@ end
...
+		if love.event then
+			love.event.pump()
+			for name, a,b,c,d,e,f in love.event.poll() do
+				if name == "quit" then
+					if not love.quit or not love.quit() or not Scene.quit() then
+						return a or 0
+					end
+				end
+                love.handlers[name](a,b,c,d,e,f)
+                Scene[name](a,b,c,d,e,f) -- handle scene event, if any
```
Comment (arthurferrai) [inline main.lua]: Please fix the indentation to match the rest of the file

File: main.lua [hunk 2]
```diff
@@ -8,60 +8,45 @@ end
...
+			end
+		end
+ 
+		-- Update dt, as we'll be passing it to update
+		if love.timer then dt = love.timer.step() end
+ 
+		-- Call update and draw
+		if love.update then love.update(dt) end -- will pass 0 if love.timer is disabled
+        Scene.update(dt)
```
Comment (arthurferrai) [inline main.lua]: Please fix the indentation to match the rest of the file

File: main.lua [hunk 3]
```diff
@@ -8,60 +8,45 @@ end
...
+
+		if love.graphics and love.graphics.isActive() then
+			love.graphics.origin()
+			love.graphics.clear(love.graphics.getBackgroundColor())
+ 
+			if love.draw then love.draw() end
+            Scene.draw()
```
Comment (arthurferrai) [inline main.lua]: Please fix the indentation to match the rest of the file

**Review by arthurferrai — CHANGES_REQUESTED** [review]
Please fix the indentation on commented lines

**Comment by arthurferrai** [comment]
thanks for pull request :D

**Comment by Veslo5** [comment]
> 
> 
> thanks for pull request :D

Np, thanks for great library! :)

**Pull request merged**